### DISCUSSION TITLE
Font awesome markers

### DISCUSSION THREAD

**Discussion by oyvindsolstad** [javalent/obsidian-leaflet#99] [Q&A] [answered]
I can't figure out how to use other markers - using font awesome.

The docs say:

_A note specified in this manner should have the following frontmatter tags: 
location: [lat, long] REQUIRED 
mapmarker: <marker-type> OPTIONAL_

**But how?**

I have a map where markers are from all notes linked FROM this note, using the **linksFrom** command.

The note has links both to states AND capitols.

On the individual notes, I'm using location in frontmatter to specify. Example for Oslo:

`location: [59.916972809425545, 10.75175617746631]`

What I would like to do is to use the [<city>](https://fontawesome.com/v5.15/icons/city?style=solid) icon in Font Awesome in the front matter for cities, and the [<flag>](https://fontawesome.com/v5.15/icons/flag?style=solid) icon for states, so that they stand apart on the map.

Is this possible? How do I write the code in frontmatter on the individual notes? I've tested these:

`location: [59.916972809425545, 10.75175617746631]
mapmarker: <city>`

`location: [59.916972809425545, 10.75175617746631]
mapmarker: city`

`location: [59.916972809425545, 10.75175617746631]
mapmarker: [city]`

`location: [59.916972809425545, 10.75175617746631]
mapmarker: ["city"]`

…but can't make it work. What am I doing wrong? Or is this possible at all?

**Comment by valentine195** [comment, marked as answer]
You can create additional marker "types" in the settings tab. You then use the name you gave the marker type in `mapmarker`.

**Reply by oyvindsolstad** [reply]
**Yey! Thanks!** 

And the way to specify was like this:

`mapmarker: [city]`

Thank you! This plugin really makes Obsidian an amazing tool.

**Reply by valentine195** [reply]
`mapmarker: city` should also work 😁

**Reply by oyvindsolstad** [reply]
It sure did! 😄
